common/api/ecschema-metadata.api.md

```diff
@@ -420,7 +420,11 @@ export const Diagnostics: {
             readonly diagnosticType: DiagnosticType;
             ecDefinition: Schema;
             messageArgs?: [string, string] | undefined;
-            category: DiagnosticCategory;
+            category: DiagnosticCategory; /**

```

Comment:
I am getting API changes in ecshema-metadata even though I have not made any changes to the API. I have run rush clean and rebuild before extract-api. These changes were also present in master. Any ideas on how to fix this?